packages/react-app/src/components/SiteFooter.jsx

```diff
@@ -12,17 +13,27 @@ const SiteFooter = () => {
         <HStack alignItems="center" spacing="7px">
           <GithubFilled />{" "}
           <Link color={linkColor} href="https://github.com/BuidlGuidl/SpeedRunEthereum" isExternal>
-            Fork me
+            <FormattedMessage id="footer.fork-me" defaultMessage="Fork me" />
           </Link>
         </HStack>
         <p>|</p>
         <HStack alignItems="center" spacing="5px">
-          <HStack alignItems="center">
-            <span>Built with</span> <HeartFilled /> <span>at</span>
-          </HStack>
-          <Link color={linkColor} href="https://buidlguidl.com/" isExternal>
-            BuidlGuidl
-          </Link>
+          <FormattedMessage
+            id="footer.built-with-love-at-buidlguidl"
+            defaultMessage="Built with {heartIcon} at <Link>BuidlGuidl</Link>"
+            values={{
+              heartIcon: (
+                <HStack alignItems="center" mx="5px">
+                  <HeartFilled />
+                </HStack>
+              ),
+              Link: chunks => (
+                <Link color={linkColor} href="https://buidlguidl.com/" isExternal>
+                  {chunks}
+                </Link>
+              ),
+            }}
+          />
```

Comment:
Here I made a change to the html structure of the footer. I think the visual result is exactly the same, but we might want to double check it. The reason is that otherwise the translation string would have ended up being:
```
<HStack><span>Built with</span> {heartIcon} <span>at</span></Hstack><Link>BuidlGuidl</Link>
```

That html is complex enough that I don't think a typical translator could re-create it in case the order of the phrase changed in a different language.